rcl/include/rcl/graph.h

```diff
@@ -430,6 +429,7 @@ rcl_names_and_types_fini(rcl_names_and_types_t * names_and_types);
  * \param[out] node_namesspaces struct storing discovered node namespaces
  * \return `RCL_RET_OK` if the query was successful, or
  * \return `RCL_RET_BAD_ALLOC` if an error occurred while allocating memory, or
+ * \return `RCL_RET_INVALID_ARGUMENT` if any arguments are invalid, or
```

Comment:
We should also expand this to include the return values from when node and namespace checking fail.

Reply:
Done 8db9fac